frontend/src/components/common/SectionFilterHeader.tsx

```diff
@@ -110,52 +110,54 @@ export default function SectionFilterHeader(props: SectionFilterHeaderProps) {
     actions.push(...preRenderFromFilterActions);
   }
 
-  if (!showFilters.show) {
-    actions.push(
-      <IconButton
-        aria-label={t('Show filter')}
-        onClick={() => setShowFilters({ show: true, userTriggered: true })}
-        size="medium"
-      >
-        <Icon icon="mdi:filter-variant" />
-      </IconButton>
-    );
-  } else {
-    actions.push(
-      <Grid container alignItems="flex-end" justifyContent="flex-end" spacing={1} wrap="nowrap">
-        {!noNamespaceFilter && (
+  if (!noSearch) {
```

Comment:
I thought we had already this capability. If this is needed in the end, please split it into its own commit.

Reply:
split into two commits